proto/api/v2/compliance_profile_service.proto

```diff
@@ -54,10 +78,17 @@ service ComplianceProfileService {
         };
     }
 
-    // ListComplianceProfiles returns profiles matching given query
-    rpc ListComplianceProfiles (RawQuery) returns (ListComplianceProfilesResponse) {
+    // ListComplianceProfiles returns profiles matching given request
+    rpc ListComplianceProfiles (ProfilesForClusterRequest) returns (ListComplianceProfilesResponse) {
+        option (google.api.http) = {
+            get: "/v2/compliance/cluster/profiles"
+        };
+    }
+
+    // ListProfileSummaries returns profiles matching each cluster and the given query
+    rpc ListProfileSummaries (ClustersProfileSummaryRequest) returns (ListComplianceProfileSummaryResponse) {
         option (google.api.http) = {
-            get: "/v2/compliance/profiles"
+            get: "/v2/compliance/summary/profiles"
```

Comment:
Do we want to put `summary` on the end of the path like we've done with other APIs where the verb is at the end?

Reply:
+1

Reply:
Took a closer look.  That clashed with `profiles/{id}`.  But of course `profiles` doesn't make sense for an id.  So I changed that to `profile/{id}` and then will flip the words on the rest.